extension/jdtls.ext/com.microsoft.gradle.bs.importer/src/com/microsoft/gradle/bs/importer/Utils.java

```diff
@@ -214,4 +221,44 @@ public static void sendTelemetry(JavaLanguageClient client, Object message) {
     client.sendNotification(new ExecuteCommandParams("_java.gradle.buildServer.sendTelemetry",
         Arrays.asList(message)));
   }
+
+  /**
+   * Extracts the jar file from the given artifact.
+  */
+  public static File getJarFile(File file) {
+
+    String filepath = file.getAbsolutePath();
+
+    if (filepath.endsWith(".aar")) {
+
+      // Extracting classes.jar from AAR files
+      try(ZipInputStream is = new ZipInputStream(new FileInputStream(file))) {
+
+        ZipEntry entry;
+        while ((entry = is.getNextEntry()) != null) {
+          if (entry.getName().equals("classes.jar")) {
+            String fileName = file.getName();
+            fileName = fileName.substring(0, fileName.length() - 4);
+            fileName = fileName + ".jar";
+            File outputFile = Path.of(file.getParentFile().getAbsolutePath(), fileName).toFile();
+            try (FileOutputStream outputStream = new FileOutputStream(outputFile)) {
+              byte[] buffer = new byte[1024];
+              int len;
+              while((len = is.read(buffer)) > 0) {
+                outputStream.write(buffer, 0, len);
+              }
+              return outputFile;
+            }
+          }
+        }
+
+      } catch(IOException e) {
+      }
```

Comment:
Please add log in the catch block.

JavaLanguageServerPlugin.logException(e);

Reply:
Addressed in https://github.com/microsoft/vscode-gradle/pull/1594/commits/4f6265177d1904c5c2a49da8347b1b2d547b0327